form/form.go

```diff
@@ -531,103 +535,101 @@ func BuildField(fileDescriptorSet *descriptor.FileDescriptorSet, msg *descriptor
 			s += '<a href="#" class="add-child btn btn-success btn-sm" role="button" type="` + typName + `">add ` + fieldname + `</a>` + tooltip + `';
 			s += '<div class="field form-group"></div>';
 			`
-		}
-	} else {
-		if !f.IsRepeated() {
-			if isBool(f) {
-				defaultBool := "\"nothing\""
-				if proto3 {
-					defaultBool = "false"
-				}
-				if f.DefaultValue != nil {
-					defaultBool = f.GetDefaultValue()
-				}
-				s := `s += '<div class="field form-group"><label class="col-sm-2 control-label">` + fieldname + tooltip + colon + ` </label>';
+	}
+	if !f.IsRepeated() {
```

Comment:
Could we keep the elses just to make this diff easier to read, for me?
We can remove them in a following commit.

Reply:
Aha, my linter told me to do this. Sorry to break the readability ;)

I will remove my commit.